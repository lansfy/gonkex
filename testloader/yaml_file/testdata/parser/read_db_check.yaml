- name: loader MUST read tests with database check
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  dbChecks:
    - dbQuery: "SELECT ALL"
      dbResponse:
        - item1
        - item2
      comparisonParams:
         ignoreArraysOrdering: true
         disallowExtraFields: true
    - dbQuery: "UPDATE ALL"
      dbResponse:
        - item3
      comparisonParams:
         ignoreValues: true
    - dbQuery: "DELETE ALL"
      dbResponse: []
