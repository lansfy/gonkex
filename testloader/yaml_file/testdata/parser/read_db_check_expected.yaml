- GetName: loader MUST read tests with database check
  GetMethod: GET
  Path: /some/path
  GetResponses:
    200: "aaabbb"
  GetDatabaseChecks:
    - DbQueryString: "SELECT ALL"
      DbResponseJson:
        - item1
        - item2
      GetComparisonParams:
        IgnoreArraysOrdering: true
        DisallowExtraFields: true
    - DbQueryString: "UPDATE ALL"
      DbResponseJson:
        - item3
      GetComparisonParams:
        IgnoreValuesChecking: true
    - DbQueryString: "DELETE ALL"
      DbResponseJson: []
  GetFileName: testdata/parser/read_db_check.yaml
  GetLineNumber: 1
  FirstTestInFile: true
  LastTestInFile: true
